example-context/example-service/example-single/pom.xml

```diff
@@ -59,6 +59,18 @@
             <version>${project.version}</version>
             <classifier>classes</classifier>
         </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.services</groupId>
+            <artifactId>event-publisher-process</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>uk.gov.justice.services</groupId>
+            <artifactId>event-publisher-process</artifactId>
+            <version>${project.version}</version>
+        </dependency>
```

Comment:
Either this has been a long code review and I am seeing double....or there is a duplicate dependency here

Reply:
Duplicate removed. Well spotted!